src/renderers/shared/fiber/__tests__/__snapshots__/ReactIncrementalPerf-test.js.snap

```diff
@@ -171,9 +171,6 @@ exports[`ReactDebugFiberPerf recovers from caught errors 1`] = `
 "// Stop on Baddie and restart from Boundary
 ⛔ (React Tree Reconciliation) Warning: There were cascading updates
   ⚛ Parent [mount]
-    ⚛ Boundary [mount]
```

Comment:
Can you explain why these disappeared in more detail? I assume this is before it has thrown.